compiler/src/test/kotlin/software/amazon/lastmile/kotlin/inject/anvil/processor/CreateComponentProcessorTest.kt

```diff
@@ -0,0 +1,207 @@
+@file:OptIn(ExperimentalCompilerApi::class)
+
+package software.amazon.lastmile.kotlin.inject.anvil.processor
+
+import assertk.assertThat
+import assertk.assertions.contains
+import com.tschuchort.compiletesting.KotlinCompilation.ExitCode.COMPILATION_ERROR
+import org.jetbrains.kotlin.compiler.plugin.ExperimentalCompilerApi
+import org.junit.jupiter.api.Test
+import software.amazon.lastmile.kotlin.inject.anvil.compile
+
+// Note that there's no unit test verifying the correctly generated code. We're blocked on testing
+// expect-actual multiplatform code in unit tests.
+//
+// https://github.com/ZacSweers/kotlin-compile-testing/issues/298
+//
+// The correctness is verified through the sample app of this project for now.
+class CreateComponentProcessorTest {
```

Comment:
Can we check the successful path? Or is there no real way to do that with `expect` / `actual`?

Reply:
No, not right now, because the compiler will pick up all generated files and then complain about that these keywords cannot be used within the same module / source set. It's a limitation of the compile testing library right now. We should help and add support for this.